scripts/process/trends_all.R

```diff
@@ -84,16 +67,20 @@ process.trends_all <- function(viz=as.viz("trends_all")){
       mutate(viewID = vid)
   }))
 
+  trends$scale[trends$scale == "Year"] <- levels_text$Year
+  trends$scale[trends$scale == "Month"] <- levels_text$Month
+  trends$scale[trends$scale == "Week"] <- levels_text$Week
+  trends$type <- factor(trends$scale, levels = levels(sessions_all$type))
+
   # augment trends with single columng for shape: is the trend up, down, or
   # non-significant?
   trends_aug <- trends %>%
-    mutate(trend = ifelse(pvalue <= 0.05, ifelse(slope > 0, "up", "down"), "none"))
+    mutate(trend = ifelse(pvalue <= 0.05, ifelse(slope > 0, viz[["up"]], viz[["down"]]), ""))
```

Comment:
want another arg for viz[["none"]]? or one length-three arg for `viz[['trend_names"]]`?